cowprotocol/cowswap
# Disable anniversary release

# Summary

Disable anniversary release

  # To Test

1. Open the page
* Should not contain the anniversary banner
2. Place an order
* Should have the regular cow sound
